RELEASE.md

```diff
@@ -18,10 +18,8 @@
          not that can be set in the project settings.
       1. `Packages > Rebuild`.
    1. Build and push the container image.
-      1. `$ buildah build -t quay.io/retis/retis:x.y.z`
-      1. `$ buildah push quay.io/retis/retis:x.y.z`
-      1. Manually tag on the web UI the image pushed with `latest`, if
-         applicable.
+      1. Manually run the workflow `build-and-push-image` selecting the branch and setting the
+         `release_tags` with the space separated list of tags (i.e. release_tags="x.y.z latest").
```

Comment:
> The current process is to build an image tagged `x.y.z` and then in the web UI to make a symlink for the `latest` tag. Wouldn't this trigger two identical builds?
> 

AFAIU, it builds once and tags it multiple times:
```
Successfully tagged localhost/retis:next
771af2b19e6e100c8ea7c6debfe675485d18aa8b63e969cc7f405b87f763b1dc
Tagging the built image with tags next,latest-next
```

Unfortunately, the scheduled run overwrote the next image, so the alias is not visible, but the history on the registry normally shows the same pattern we have with latest <-> 1.4.0

> On the other hand it makes updating the `latest` tag automatic.
> 

That was the idea explicitly supporting multiple tags.

> Also can we be a little bit more explicit on where to launch the workflow? (eg adding "in the Actions tab on the Retis Github page").

I think this will be the link, I'll add that as well
[https://github.com/retis-org/retis/actions/workflows/build_push_image.yaml](https://github.com/retis-org/retis/actions/workflows/build_push_image.yaml)